CMakeLists.txt

```diff
@@ -851,8 +867,7 @@ IF(ENABLE_OPENSSL AND NOT CMAKE_SYSTEM_NAME MATCHES "Darwin")
   FIND_PACKAGE(OpenSSL)
   IF(OPENSSL_FOUND)
     SET(HAVE_LIBCRYPTO 1)
-    INCLUDE_DIRECTORIES(${OPENSSL_INCLUDE_DIR})
```

Comment:
Do we want to get rid of this?

Reply:
For building libarchive, `OpenSSL::Crypto' will make CMake pass include dirs and link libs to the targets that link to it. So there is no reason to use both variables and targets.
For exported CMake config, using targets implies using `find_dependency` (for static linkage).